tools/debian/control

```diff
@@ -70,26 +74,26 @@ Description: Cockpit deployment and developer guide
  or extend Cockpit.
 
 Package: cockpit-docker
-Architecture: amd64 armel armhf i386
+Architecture: all
 Depends: ${misc:Depends},
          docker.io (>= 1.3.0) | docker-engine (>= 1.3.0),
-         python
+         cockpit-bridge (>= ${source:Version}),
+         cockpit-bridge (<< ${source:Version}.1~)
 Description: Cockpit user interface for Docker containers
  The Cockpit components for interacting with Docker and user interface.
 
 Package: cockpit-machines
-Architecture: any
+Architecture: all
 Depends: ${misc:Depends},
          libvirt-daemon-system
 Description: Cockpit user interface for virtual machines
  The Cockpit components for managing virtual machines.
 
 Package: cockpit-networkmanager
-Architecture: any
-# Lock bridge dependency due to --with-networkmanager-needs-root
-# which uses new less stable /manifests.js request path.
-depends: ${misc:Depends},
-         cockpit-bridge (= ${binary:Version}),
+Architecture: all
+Depends: ${misc:Depends},
+         cockpit-bridge (>= ${source:Version}),
+         cockpit-bridge (<< ${source:Version}.1~),
```

Comment:
Ah, I see what you mean, I thought you were saying that the depends isn't strict enough. Maybe we can take `required_base` out of `cockpit.spec` in the future and into something which both the spec file and debian/rules read?